scripts/docs-versions.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+set -e
+
+printf '['
+latest='"latest"'
+for ver in $(git ls-remote --heads "${1:-.}" | grep -P -o '(?<=refs/heads/)[0-9][0-9.]+' | sort -V -r); do
```

Comment:
There is the `--sort` option of `git ls-remote` that can be used.
It allows to treat tags as versions and sort them accordingly.

Reply:
I think I will not change this. I trust the current approach a lot more than this:

```suggestion
for ver in $(git ls-remote --heads --sort='-version:refname' "${1:-.}" '[0-9].*' | grep -P -o '(?<=refs/heads/)[0-9.]+'); do
```

The sorting feature is quite finicky and I don't quite understand it.

I also wonder whether I intentionally avoided it the last time I did this.